packages/mobile/src/backup/BackupForceScreen.tsx

```diff
@@ -0,0 +1,82 @@
+import Button from '@celo/react-components/components/Button'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { BackHandler, ScrollView, StyleSheet, Text } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { OnboardingEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import DelayButton from 'src/backup/DelayButton'
+import { Namespaces } from 'src/i18n'
+import Logo from 'src/icons/Logo'
+import { emptyHeader } from 'src/navigator/Headers'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+
+type Props = StackScreenProps<StackParamList, Screens.BackupForceScreen>
+
+function BackupForceScreen({ navigation }: Props) {
+  const { t } = useTranslation(Namespaces.backupKeyFlow6)
+
+  const startBackup = () => {
+    ValoraAnalytics.track(OnboardingEvents.backup_start)
+    navigate(Screens.AccountKeyEducation)
+  }
+
+  // Prevent back button on Android
+  useEffect(() => {
+    const backPressListener = () => true
+    BackHandler.addEventListener('hardwareBackPress', backPressListener)
+    return () => BackHandler.removeEventListener('hardwareBackPress', backPressListener)
+  }, [])
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView contentContainerStyle={styles.content}>
+        <Logo height={32} />
+        <Text style={styles.title}>{t('backupSetupTitle')}</Text>
+        <Text style={styles.body}>{t('backupSetupBody')}</Text>
+        <Button text={t('introPrimaryAction')} onPress={startBackup} testID="SetUpAccountKey" />
+      </ScrollView>
+    </SafeAreaView>
+  )
+}
+
+BackupForceScreen.navOptions = {
+  ...emptyHeader,
+  // Prevent swiping back on iOS
+  gestureEnabled: false,
+  headerLeft: () => null,
+  headerRight: () => <DelayButton />,
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'column',
```

Comment:
nit: this is the default already.
```suggestion
```

Reply:
Yeah, I know. I just always put the direction when it's important because it helps imo to have that sort of things explicit, but I can see how one might think it's unnecessary. Removed.

Reply:
Yes no worry, fine to keep it too 👍